x-pack/platform/plugins/shared/maps/ui_tests/tests/full_screen_mode.spec.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, tags, test } from '@kbn/scout';
+import {
+  VISIBLE_CHROME,
+  HIDDEN_CHROME,
+  FULL_SCREEN_MODE,
+  EXIT_FULL_SCREEN,
+} from '../fixtures/constants';
+
+test.describe('Maps', { tag: tags.DEPLOYMENT_AGNOSTIC }, () => {
+  test.beforeEach(async ({ browserAuth, pageObjects }) => {
+    await browserAuth.loginAsViewer();
+    await pageObjects.maps.goto();
+    await pageObjects.maps.waitForRender();
```

Comment:
I don't think `waitForRender` belongs in MapsPage. How about putting in a RenderablePage since this is a concept that most embeddables use.